specification/documentation/APIDescription.md

```diff
@@ -196,4 +196,20 @@ The channels can have the following supplier statuses:
 * `notified` - a push notification is reported as having been successfully relayed to one or more devices
 * `received` - the request has been received by the supplier and is queued to be processed
 
-Statuses for other suppliers will be added in future.
+### GOV.UK Notify (Email/SMS)
```

Comment:
I'd suggest breaking the Email and SMS into their own sections in case either or get more additional statuses if we use new suppliers for either Email or SMS

Reply:
e.g. 
```
### Email
- ... - ...

### SMS
- ... - ...

### Letter
- ... - ....

```

Reply:
Done.